packages/landing/src/styles/docs.css

```
@@ -0,0 +1,265 @@
+/* Map Starlight tokens onto the landing page's dark/mono palette.
+   Palette mirrors ProseLayout.astro :root. The site is dark-only. */
+
+@import url("https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@400;500;600&family=Lora:wght@400&display=swap");
```

Stylelint (17.13.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@400;500;600&family=Lora:wght@400&display=swap")" to be ""https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@400;500;600&family=Lora:wght@400&display=swap"" (import-notation)

(import-notation)